tests/integration/components/instance-link-test.js

```diff
@@ -0,0 +1,30 @@
+import { render } from "@ember/test-helpers";
+import { hbs } from "ember-cli-htmlbars";
+import engineResolverFor from "ember-engines/test-support/engine-resolver-for";
+import { setupRenderingTest } from "ember-qunit";
+import { module, test } from "qunit";
+
+const modulePrefix = "ember-ebau-gwr";
+const resolver = engineResolverFor(modulePrefix);
+
+module("Integration | Component | instance-link", function (hooks) {
+  setupRenderingTest(hooks, { resolver, integration: true });
+
```

Comment:
This will fail, it would be nice if we actually had a small test here i think. So if you have time... :smile:

Reply:
Added test